recipes/sherpa/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "sherpa" %}
+{% set version = "v2.2.16" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/sherpa-team/sherpa/-/archive/{{ version }}/sherpa-{{ version }}.tar.gz
+  sha256: b2680da5682c78623635ae88a5ff5af346cf74c1be95ee9177e8f9080d418e76
+  patches:
+    # Avoid issue with compiling .pyc files during packaging step post build
+    - 0001-convert-print-statements-using-2to3.patch
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
+    - {{ compiler('fortran') }}
+    - gnuconfig  # [unix]
+    - autoconf
+    - libtool
+    - make
+    - automake
+    - which
+    - sed  # needed only to patch LDFLAGS
+    - diffutils
+    - texinfo  # can't skip build of manual
+  host:
+    - sqlite
+    - lhapdf
+    - hepmc2
+  run:
+    - lhapdf
+    - hepmc2
+
+test:
+  requires:
+    - {{ compiler('cxx') }}
+    - lhapdf
+    - hepmc2
+  files:
+    - $PREFIX/bin/Sherpa
+    - $PREFIX/bin/Sherpa-config
+  source_files:
+    - Examples
+  commands:
+    - Sherpa-config
+    - Sherpa-config --version
+    - Sherpa-config --prefix
+    - Sherpa-config --incdir
+    - Sherpa-config --libs
+    - Sherpa-config --datadir
+    - Sherpa-config --ldflags
+    - Sherpa-config --cppflags
+
+    - Sherpa --help
+    - cd Examples/CI/LO_Z
+    - Sherpa
```

Comment:
At `Sherpa` runtime the error

```
Library_Loader::LoadLibrary(): $PREFIX/lib/SHERPA-MC: cannot read file data: Is a directory
Library_Loader::LoadLibrary(): Failed to load library 'libSherpaSM.so'.
Sherpa: Initialization_Handler::InitializeTheModel throws missing module: 
   Cannot load model library SherpaSM.
```

occurs but I don't understand why given the `LDFLAGS` that are used in `recipes/sherpa/build.sh`.

I'm [unable to reproduce this locally](https://gist.github.com/matthewfeickert/d949bb874a5465328d03c276a64332f7) and I can't find anything in the [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) that covers this.

@conda-forge/help-c-cpp can you offer any suggestions?